ddtrace/tracer/tracer_test.go

```diff
@@ -464,6 +464,19 @@ func TestTracerNoDebugStack(t *testing.T) {
 	assert.Empty(s.Meta[ext.ErrorStack])
 }
 
+func TestTracerNoDebugStackFinishSetTag(t *testing.T) {
+	assert := assert.New(t)
+	tracer := newTracer(WithDebugStack(false))
+	s := tracer.StartSpan("web.request").(*span)
+	err := errors.New("error value with no trace")
+	s.SetTag(ext.Error, err)
+
+	assert.Equal(int32(1), s.Error)
+	assert.Equal("error value with no trace", s.Meta[ext.ErrorMsg])
+	assert.Equal("*errors.errorString", s.Meta[ext.ErrorType])
```

Comment:
You can remove these lines, we are strictly testing the stack trace here, not that setting error tags works. That's tested elsewhere. Let's keep tests focused.